### DISCUSSION TITLE
How to use pinia in Quasar framework?

### DISCUSSION THREAD

**Discussion by donPuerto** [quasarframework/quasar#9212] [General - Components / Directives / etc] [answered]
Im having error in boot

```js
import { boot } from 'quasar/wrappers';
import { createPinia } from 'pinia'


export default boot(({app}) => {
    const pinia = createPinia()
    app.use(pinia)
    console.log('Pinia was created', pinia)

});

export { pinia }
```

![image](https://user-images.githubusercontent.com/9329005/117592253-786d2280-b16a-11eb-91f0-4cf2ab7cf793.png)

**Comment by metalsadman** [comment, marked as answer]
because your 'pinia' variable is out of scope. 

could you please delete your duplicated thread https://github.com/quasarframework/quasar/discussions/9209.

**Reply by donPuerto** [reply]
I cant delete the thread
